services/osparc-gateway-server/src/osparc_gateway_server/backend/utils.py

```diff
@@ -283,3 +284,31 @@ async def create_docker_secrets_from_tls_certs_for_cluster(
             secret_data=cluster.tls_key.decode(),
         ),
     ]
+
+
+OSPARC_SCHEDULER_PORT: Final[int] = 8786
+
+
+def get_osparc_scheduler_cmd_modifications(
+    scheduler_service_name: str,
+) -> Dict[str, str]:
+    # NOTE: the healthcheck of itisfoundation/dask-sidecar expects the dashboard
+    # to be on port 8787
+    # (see https://github.com/ITISFoundation/osparc-simcore/blob/f3d98dccdae665d23701b0db4ee917364a0fbd99/services/dask-sidecar/Dockerfile)
+    return {
+        "--dashboard-address": ":8787",
```

Comment:
minor: am I mistaken to suggest this?
```suggestion
        "--dashboard-address": f":{OSPARC_SCHEDULER_PORT}",
```

Reply:
the suggestion is wrong, but I will create a variable.
this port is the scheduler dashboard port